docs/crocks/Identity.md

```diff
@@ -0,0 +1,61 @@
+# Identity
+
+`Identity a`
+
+Crock which returns the same value that was used as its argument
+
+## Why?
+
+To expose the constructor and instances defined below
+
+| Constructor | Instance |
+|:---|:---|
+| [`of`](#of) | [`ap`](#ap), [`chain`](#chain), [`equals`](#equals), [`map`](#map), [`of`](#of), [`sequence`](#sequence), [`traverse`](#traverse), [`value`](#value) |
+
+## Constructors
+
+### of
+
+`Identity a => a -> Identity a`
+
+## Instances
+
+### inspect
+
+`Identity a => Identity a`
+
+### value
+
+`() -> a`
+
+### type
+
+`() -> Identity`
+
+### equals
+
+`a -> Boolean`
+
+### map
+
+`Identity m => m (a -> b) -> m b`
```

Comment:
So `map` takes a function and lifts it into the context of `Identity`, and because we need to show how the mapping works, we will want to use `~>` here.  So this is how it should be shown IMO:
`Identity m => m a ~> (a -> b) -> m b`